release-notes.txt

```diff
@@ -1,3 +1,548 @@
+* cord_lc: Process truncated reads (#20547)
+* gnrc_ipv6_nib: Ignore PIO with on-link flag (#20418)
+* nanocoap/cache: Extend with option-only cache keygen (#20043)
+* pkg/tinydtls: enforce the default dtls user params to be configurable (#20478)
+* routing/rpl: Fixing styling & format (#20294)
+* sys/net/gnrc: Remove code duplication (#16502)
+
+Packages
+--------
+
++ pkg/wakaama: add Light Control object implementation (#16204)
+* ci: support git-cache-rs (#20311)
+* Makefile.{include,base},pkg/nanopb: fix target deps & code generation (#20400)
+* pkg/flashdb: enhance FAL config (#20221)
+* pkg/uzlib: warn about memory leak during compression (#20293)
+
+Boards
+------
+
++ boards/cpu: Add GBA boot support to RIOT (#19519)
++ boards/nucleo-c031c6: add PWM configuration (#20520)
++ boards/nucleo-u575zi-q: Add support (#20444)
++ boards/weact-g030f6: add support for minimal break-out board (#20337)
++ cpu/rpx0xx: add initial pwm support (#20473)
++ cpu/stm32: add support for STM32C0 and NUCLEO-C031C6 (#20300)
+* boards/feather-nrf52840-sense: declare lsm6ds33 accelerometer (#20436)
+* boards/feather-nrf52840-sense: use generic lsm6dsxx (#20526)
+* cpu/msp430/f2xx: clean up periph_uart,periph_spi (#20357)
+* cpu/native: Enable Rust on 64bit (#20339)
+* cpu/nrf9160: fix GPIOTE ISR (#20560)
+* cpu/samd5x write can driver (#19736)
+* cpu/stm32u5: Enable rtc support (#20521)
+* drivers/lsm6dsxx: refactoring Lsm6dsl into common driver Lsm6dsxx (#20170)
+* native64: Separate board for 64-bit native (#20335)
+* native: Linux/x86_64 support (#20315)
+
+CPU
+---
+
++ cpu/native: add gpio-mock (#20431)
++ cpu/native: Add support for periph_timer_query_freqs (#20306)
++ drivers/periph_gpio_ll: Implement API to switch direction (#20292)
++ stdio_dispatch: allow to select multiple stdio methods at the same
+  time (#19738)
+* cpu/esp8266: allow arbitrary SPI clocks (#20410)
+* cpu/native/periph/uart: uart_poweroff() closes the file descriptor +
+  extended baud-rate support on Linux (#20365)
+* cpu/native: Fix executable stack warning (#20307)
+* cpu/native: return error code on failed assertion (#20419)
+* cpu/stm32/periph/gpio: simplify condition code (#20479)
+* cpu/stm32/periph/usbdev_fs: avoid using ztimer when not needed (#20467)
+* cpu/stm32: Fix ifdefs in periph_gpio_ll (#20530)
+* cpu/stm32: remove redundant conditional compile block (#20495)
+* drivers/mtd_flashpage: allow to define AUX slot on flash (#18608)
+* sys/byteorder: clean up implementation (#20313)
+* sys/cpp_new_delete: always enable the module when C++ is used (#20348)
+* tests: exit `native` with error value on failure (#20445)
+* treewide: Rust updates (#20303)
+
+Device Drivers
+--------------
+
+* driver/lsm6dsxx: Allow generic LSM6DSXX driver (#20504)
+* drivers/w5100: make use of netdev_eui48_get() (#20304)
+
+Documentation
+-------------
+
++ build system: add list of features for documentation and sanity
+  checking (#20366)
++ doc: add graphic to vision page (#20498)
```

Comment:
I think this could be merged with `doxygen: added vision.md imported from Wiki (#20491)`.